design/20220118.server-side-apply.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Server Side Apply
+authors:
+  - "@joshvanl"
+reviewers:
+  - @jetstack/team-cert-manager
+approvers:
+  - @jetstack/team-cert-manager
+editor: "@joshvanl"
+creation-date: 2022-01-18
+last-updated: 2022-01-18
+status: implementable
+---
+
+# Server Side Apply
+
+## Table of Contents
+
+<!-- toc -->
+- [Summary](#summary)
+  * [Conflicts](#conflicts)
+  * [Deleting Fields](#deleting-fields)
+- [Field Manager and User Agent](#field-manager-and-user-agent)
+    + [Known Field Managers](#known-field-managers)
+  * [Scheme](#scheme)
+  * [Field Manager](#field-manager)
+  * [User Agent](#user-agent)
+- [Implementation Consideration](#implementation-consideration)
+  * [API Call Code Changes](#api-call-code-changes)
+  * [Force parameter](#force-parameter)
+  * [client-go Testing](#client-go-testing)
+  * [API Changes](#api-changes)
+- [Feature Gate](#feature-gate)
+- [Migration](#migration)
+<!-- /toc -->
+
+## Summary
+
+Server Side Apply is a [Kubernetes
+feature](https://kubernetes.io/docs/reference/using-api/server-side-apply/)
```

Comment:
tiny nit: it seems like the Kubernetes documentation and the KEP [555-server-side-apply.md](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/555-server-side-apply/README.md#server-side-apply) are using the typography "Server-Side Apply"

```suggestion
Server-Side Apply is a [Kubernetes
feature](https://kubernetes.io/docs/reference/using-api/server-side-apply/)
```

Reply:
Fixed in https://github.com/jetstack/cert-manager/pull/4758/commits/bfcaa3dd55c7f7c8e98c9775dfd12fbb49d514aa